static/templates/settings/organization-profile-admin.handlebars

```diff
@@ -43,5 +44,17 @@
                         id="realm_icon_delete_button">{{t 'Delete icon' }}</button>
             </div>
         </div>
+
+        {{#if is_admin}}
```

Comment:
I don't think we want the if is_admin here; in general we want to show all the settings to everyone, and just have the button be disabled.